editor/opening_hours_ui.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include "std/set.hpp"
+#include "std/vector.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace editor
+{
+namespace ui
+{
+
+using TOpeningDays = set<osmoh::Weekday>;
```

Comment:
Okay, seems that we will have the same problem even after replace of using to class declaration. OK, leave this as is.